README.md

```diff
@@ -1,5 +1,9 @@
 # CP2K
 
+[![Release Status][release-badge]][release-link]
+[![Fedora Status][fedora-badge]][fedora-link]
```

Comment:
Interesting that the pre-commit suggests compressing these lines. I don't think that's a good thing to do in this case. It creates unnecessary changes when lines are added/removed, and it is less readable

Reply:
Maybe we need to [update](https://github.com/cp2k/cp2k/blob/master/tools/precommit/requirements.txt) our mdformat and perhaps also install the `mdformat-footnote` plugin? I can prepare a PR for that.

Reply:
Not really, it just so happens that I am hitting `--wrap=100` which is not set on `spglib`. I've made an issue upstream about it: https://github.com/executablebooks/mdformat/issues/437. I am not sure how to design it on their side, so I will probably keep this on hold and wrap according to pre-commit for now.